hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -306,7 +306,7 @@ objects:
           nodeAffinity:
             preferredDuringSchedulingIgnoredDuringExecution:
             - preference:
-              matchExpressions:
+                matchExpressions:
```

Comment:
What happened here for this change?

Reply:
I'm not sure?  I ran `make` and found that.

Reply:
I'm not sure?  I ran `make` and found this.

Reply:
Spooky.